<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsWPilotRaDcSpeedDelta</class>
 <widget class="QWidget" name="tlinsWPilotRaDcSpeedDelta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Axis</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioHourAngle">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string> Hour angle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDeclination">
        <property name="text">
         <string> Declination</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupHourAngle">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Hour angle</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="tlinsWCoordinateHourAngle" name="valueHourAngle" native="true">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[hours/s]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="negative">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Negative</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDeclination">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Declination</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="tlinsWCoordinateDeclination" name="valueDeclination" native="true">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[degris/s]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tlinsWCoordinateHourAngle</class>
   <extends>QWidget</extends>
   <header>tlinswcoordinatehourangle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tlinsWCoordinateDeclination</class>
   <extends>QWidget</extends>
   <header>tlinswcoordinatedeclination.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioHourAngle</sender>
   <signal>clicked()</signal>
   <receiver>tlinsWPilotRaDcSpeedDelta</receiver>
   <slot>raSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioDeclination</sender>
   <signal>clicked()</signal>
   <receiver>tlinsWPilotRaDcSpeedDelta</receiver>
   <slot>dcSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>raSelected()</slot>
  <slot>dcSelected()</slot>
 </slots>
</ui>
